frontend/src/modules/machinery/components/TranslationSource.js

```diff
@@ -51,5 +68,31 @@ export default function TranslationSource({ translation, locale }: Props) {
         }
     });
 
-    return <ul className='sources'>{translationSource}</ul>;
+    const isConcordanceSearch = 'projectNames' in translation;
+    const getProjectNames = () => {
+        if (!translation.projectNames) {
+            return;
+        }
+        if (translation.projectNames.every((projectName) => !projectName)) {
+            return 'Translation Memory';
+        }
```

Comment:
You'd need to make that localizable... but we'll only show "Translation Memory" alone (if there aren't any projects), so I'd instead either remove the tooltip or probably even better - reuse the `TranslationMemory` component.